tests/testthat/test-plot_tree.R

```diff
@@ -1,3 +1,13 @@
+get_default_filepaths <- function(dir_path, base_name) {
+  filepaths <- c(
+    noninteractive = file.path(dir_path, glue::glue("{base_name}.svg")),
+    interactive_html = file.path(dir_path, glue::glue("{base_name}.html")),
+    interactive_rds = file.path(dir_path, glue::glue("{base_name}.rds"))
+  )
+
+  filepaths
+}
+
 describe("save_trees", {
```

Comment:
These tests have been constructed so that they look virtually identical. They could reasonably be rewritten as a single testthat block parameterised by the different arguments to `save_trees` using {patrick}. Adding the extra dependency seemed over the top to reduce 4 simple tests to 1, though.

Reply:
Happy to have a little duplication to avoid extra dependencies